packages/puppeteer-parse/index.js

```diff
@@ -722,7 +744,7 @@ async function retrieveHtml(page) {
           }
         })();
       }),
-      page.waitForTimeout(5000), //5 second timeout
+      await page.waitForTimeout(1000), //5 second timeout
```

Comment:
I think this comment might be inaccurate now.

Reply:
updated